### PR TITLE
fix: hide browser wallets for mobile and tablets

### PR DESCRIPTION
**Pull request by Abhikumar98**
## When the website is opened in desktop view:

CleanShot 2023-09-15 at 18 26 50@2x

## When the website is opened in mobile/responsive view:
CleanShot 2023-09-15 at 18 26 35@2x

The requirement is to hide the browser wallets and show users the correct option for connecting their wallets via walletconnect.

### CODE REVIEW

**Comment by Abhikumar98** [comment]
# ⚠️ Do not merge/approve this PR

There is an issue going on with walletconnect on our end (ig), and the resolution of this ticket heavily depends upon Walletconnect.



https://github.com/lifinance/widget/assets/35174288/f2906620-ecdf-4c8c-9473-024ed475c823

File: packages/widget/src/pages/SelectWalletPage/SelectWalletPage.tsx [hunk 1]
```diff
@@ -18,6 +20,8 @@ import { ListItemText } from '../../components/ListItemText';
 import { useNavigateBack } from '../../hooks';
 import { useWallet } from '../../providers';
 
+const defaultTabWidth = 892;
```
Comment (chybisov) [inline packages/widget/src/pages/SelectWalletPage/SelectWalletPage.tsx:23]: Let's reduce it to a more mobile-centric value like the default `sm` breakpoint.

**Review by chybisov — CHANGES_REQUESTED** [review]

File: packages/widget/src/pages/SelectWalletPage/SelectWalletPage.tsx (hunk 1 above)
Comment (Abhikumar98) [reply packages/widget/src/pages/SelectWalletPage/SelectWalletPage.tsx:23]: Sure, we don't want to consider any tablet users?

Comment (chybisov) [reply packages/widget/src/pages/SelectWalletPage/SelectWalletPage.tsx:23]: Nice question, tablets resolution can be high to identify it as a tablet or not correctly. They can still select WalletConnect, there are no restrictions on that.

Comment (Abhikumar98) [reply packages/widget/src/pages/SelectWalletPage/SelectWalletPage.tsx:23]: Ahh got it. Makes sense

**Review by chybisov — APPROVED** [review]

**Comment by juscamarena** [comment]
@chybisov it seems when embedded in a mobile app I don't see this change being triggered, any idea?
wc

**Pull request merged**